content/docs/data-fetching/custom-queries.md

```diff
@@ -33,12 +39,16 @@ query postWithNav($relativePath: String!) {
 }
 ```
 
+It is good practice to add a [`graphql.config.js`](https://github.com/tinacms/tina-cloud-starter/blob/main/graphql.config.js) that points to `.tina/__generated__/schema.gql` and `.tina/__generated__/frags.gql`.
```

Comment:
minor thing, but I'm wondering why it is good practice

Reply:
For autocompletion in the code editor (VS code). Ill update the doc.